cloud/deploys/staging_azure/staging.tfvars

```diff
@@ -1,8 +1,6 @@
 postgres_admin_login = "psqladmin"
 
 application_name       = "sgtest"
-docker_repository_name = "civiform"
```

Comment:
i'd slightly prefer that we do this in your next PR cuz won't this break the deploy currently (or am i missing something)

Reply:
yeah this will break the current deploy. i recommend merging #2045 first